logging/cluster-logging-release-notes.adoc

```diff
@@ -171,7 +290,19 @@ From {product-title} 4.6 to the present, forwarding logs by using the following
 
 Instead, use the following non-legacy methods:
 
-* xref:../logging/cluster-logging-external.adoc#cluster-logging-collector-log-forward-fluentd_cluster-logging-external[Forwarding logs using the Fluentd forward protocol]
+* xref:../logging/cluster-logging-external.adoc#cluster-logging-collector-log-forward-fluentd_cluster-logging-external[Forwarding logs using the Fluentd fohttps://www.redhat.com/security/data/cve/CVE-2021-22922.htmlrward protocol]
```

Comment:
@libander It looks like the URL for a CVE was accidentally pasted into the link text.
```suggestion
* xref:../logging/cluster-logging-external.adoc#cluster-logging-collector-log-forward-fluentd_cluster-logging-external[Forwarding logs using the Fluentd forward protocol]
```